compose/config.py

```diff
@@ -63,12 +69,57 @@
 }
 
 
-def load(filename):
-    working_dir = os.path.dirname(filename)
-    return from_dictionary(load_yaml(filename), working_dir=working_dir, filename=filename)
+SUPPORTED_FILENAMES = [
+    'docker-compose.yml',
+    'docker-compose.yaml',
+    'fig.yml',
+    'fig.yaml',
+]
+
+
+log = logging.getLogger(__name__)
+
+
+ConfigDetails = namedtuple('ConfigDetails', 'config working_dir filename')
+
+
+def find(base_dir, filename):
+    if filename == '-':
+        return ConfigDetails(yaml.safe_load(sys.stdin), os.getcwd(), None)
+
+    filename = get_config_path(base_dir, filename)
+    return ConfigDetails(load_yaml(filename), os.path.dirname(filename), filename)
+
 
+def get_config_path(base_dir, file_path):
+    if file_path:
+        return os.path.join(base_dir, file_path)
```

Comment:
I had just copied the old `get_config_path()` from `Command`, but now that we have another place to move it, I think that makes sense.